docs/build/guides/transactions/create-account.mdx

```diff
@@ -11,278 +11,829 @@ import { Alert } from "@site/src/components/Alert";
 
-[Accounts](../../../learn/fundamentals/stellar-data-structures/accounts.mdx) are a fundamental building block of Stellar: they hold all your balances, allow you to send and receive payments, and let you place offers to buy and sell assets. Since pretty much everything on Stellar is in some way tied to an account, the first thing you generally need to do when you start developing is create one. This beginner-level tutorial will show you how to do that.
+[Accounts](../../../learn/fundamentals/stellar-data-structures/accounts.mdx) are a fundamental building block of Stellar: they hold all your balances, allow you to send and receive payments, and let you place offers to buy and sell assets. Since pretty much everything on Stellar is in some way tied to an account, the first thing you generally need to do when you start developing is create one. This beginner-level tutorial walks through the four building blocks you'll need: generating keys, funding an account, creating another account, and fetching balances.
 
 ## Create a Keypair
 
-Stellar uses public key cryptography to ensure that every transaction is secure: every Stellar account has a keypair consisting of a **public key** and a **secret key**. The public key is always safe to share — other people need it to identify your account and verify that you authorized a transaction. It's like an email address. The secret key, however, is private information that proves you own — and gives you access to — your account. It's like a password, and you should never share it with anyone.
+Stellar uses public key cryptography to secure every transaction: each Stellar account has a keypair consisting of a **public key** and a **secret key**. The public key is always safe to share — other people need it to identify your account and verify that you authorized a transaction. It's like an email address. The secret key, however, is private information that proves you own — and gives you access to — your account. It's like a password, and you should never share it with anyone.
 
 Before creating an account, you need to generate your own keypair:
 
 <CodeExample>
 
 ```js
+import { Keypair } from "@stellar/stellar-sdk";
+
 // create a completely new and unique pair of keys
 // see more about KeyPair objects: https://stellar.github.io/js-stellar-sdk/Keypair.html
-const pair = StellarSdk.Keypair.random();
+const parent = Keypair.random();
 
-pair.secret();
-// SAV76USXIJOBMEQXPANUOQM6F5LIOTLPDIDVRJBFFE2MDJXG24TAPUU7
-pair.publicKey();
-// GCFXHS4GXL6BVUCXBWXGTITROWLVYXQKQLF4YH5O5JT3YZXCYPAFBJZB
+console.log("Secret:", parent.secret());
+console.log("Public:", parent.publicKey());
 ```
 
 ```java
 // create a completely new and unique pair of keys.
 // see more about KeyPair objects: https://stellar.github.io/java-stellar-sdk/org/stellar/sdk/KeyPair.html
```

Comment:
this link never worked. it also highlights awareness at this point to change any ref's to java sdk repo to be - https://github.com/lightsail-network/java-stellar-sdk